packages/app/tailwind.config.js

```diff
@@ -1,5 +1,11 @@
 const colors = require('tailwindcss/colors');
 
+delete colors['lightBlue'];
```

Comment:
why this is needed?

Reply:
@pedronauck implemented it to have less dev logs in CI